cnf-tests/oslat-run.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# env vars:
+# RTPRIO (default to 1): set the SCHED_FIFO priority
+# RUNTIME_SECONDS (default 10 minute): how long the test will run
+# LOG_DIR (default stdout): the log directory where to save the ouput of the oslat command
+
+RTPRIO=${RTPRIO:-1}
+RUNTIME_SECONDS=${RUNTIME_SECONDS:-600}
+
+cpulist=()
+for cpu in $(cat /proc/self/status | grep Cpus_allowed_list: | cut -f 2 | awk '/-/{for (i=$1; i<=$2; i++)printf "%s%s",i,ORS;next} 1' RS=, FS=-)
```

Comment:
what if the container is assigned a non-consecutive block of cpus? like `0,2-4`

Can something like this turn handy?
https://github.com/fromanirh/numalign/tree/master/cmd/splitcpulist

Reply:
It works as expected, I checked it for the number of inputs, but it definitely can be much better to re-write this script on go, if you insist I can do under the current PR, or I will do it under the separate one

Reply:
No, I'm not insisting at all. We should eventually rewrite this to make it more robust, but not now.
We can keep this PR as it is.